### PR TITLE
docs: update function description in `math/base/special/log`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   updates the function description in [`index.js`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/log/lib/index.js#L22) and [`package.json`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/log/package.json#L4) to match with that in [`main.js`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/log/lib/main.js#L29) and everywhere else.

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Review by Planeshifter — COMMENTED** [review]
Wondering whether we should change the description to

> Compute the base `b` logarithm of a double-precision floating-point number.

everywhere for this package instead (assuming we may want to add `logf` down the road...)

**Comment by gunjjoshi** [comment]
@Planeshifter Yes, this caught my eye as I was working on `logf`. I'll change it according to your suggestion, everywhere in `log` and future `logf`.

File: lib/node_modules/@stdlib/math/base/special/log/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 # Logarithm
 
-> Compute the base `b` [logarithm][logarithm].
+> Computes the base `b` [logarithm][logarithm] of a double-precision floating-point number.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/log/README.md]:
```suggestion
> Compute the base `b` [logarithm][logarithm] of a double-precision floating-point number.
```

**Review by Planeshifter — APPROVED** [review]
Thanks, @gunjjoshi!

**Pull request merged**